scss/tokens/_raw.scss

```diff
@@ -17,3 +17,34 @@ $ouds-opacity-700: .64 !default;
 // $ouds-opacity-800: .80 !default;
 $ouds-opacity-900: 1 !default;
 // scss-docs-end ouds-opacities-variables
+
+// Elevation
+
+// scss-docs-start ouds-raw-shadow
```

Comment:
Should probably be `ouds-raw-elevation`. At least replacing "shadow" by "elevation".